chapter06/CliffWalking.py

```diff
@@ -83,7 +83,7 @@ def sarsa(stateActionValues, expected=False, stepSize=ALPHA):
         else:
             # calculate the expected value of new state
             valueTarget = 0.0
-            bestActions = argmax(stateActionValues[newState[0], newState[1], :], unique=False)
+            bestActions = np.argmax(stateActionValues[newState[0], newState[1], :], unique=False)
```

Comment:
`np.argmax` will return the index of the first maximum items, while `argmax` in `utils` will return a random one.